tests/unit/test_endpoint.py

```diff
@@ -140,27 +140,6 @@ def setUp(self):
         super(TestRetryInterface, self).setUp()
         self.retried_on_exception = None
 
-    def max_attempts_retry_handler(self, attempts, **kwargs):
```

Comment:
Sorry, meant to add a note about that.  This was part of the original PR in #965.  It doesn't look like the tests use these methods anymore but they were never removed.

I can also add them back and send a general "test_endpoint.py" cleanup PR if it helps.